### PR TITLE
Update column-formatting.md

### PR DESCRIPTION
**Pull request by ldemaris**
| Q                   | A
| ---------------     | ---
| content fix?        | no - yes?
| New article?        | no - yes?
| Related issues?     | fixes #X, partially #Y, mentioned in #Z

#### What's in this Pull Request?

Please describe the changes in this PR. Sample description or details around bugs which are being fixed.

#### Guidance
*You can delete this section when you are submitting the pull request.* 
* *Please update this PR information accordingly. We'll use this as part of our release notes in monthly communications.*
* *Please target your PR to 'master' branch. Released documents are in live branch.*

### CODE REVIEW

**Comment by spdevdocs** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/declarative-customization/column-formatting.md](https://github.com/ldemaris/sp-dev-docs/blob/patch-1/docs/declarative-customization/column-formatting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C13%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C201710130207282813-930%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Z03DjT18PRdmdlnoza7oGv5FrFbPbovHhQvHIvXPgoY%3D&st=2017-10-13T02%3A03%3A25Z&se=2017-11-13T02%3A08%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by spdevdocs** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/declarative-customization/column-formatting.md](https://github.com/ldemaris/sp-dev-docs/blob/patch-1/docs/declarative-customization/column-formatting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C13%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C201710130416322445-930%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OWei4LylFE%2Bq7EGstolCs8ToAPZAVn9pBZxV3USCeEo%3D&st=2017-10-13T04%3A13%3A19Z&se=2017-11-13T04%3A18%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/declarative-customization/column-formatting.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The following table compares column formatting and the Field Customizer.
 
 | Field type        | Column formatting          | Field Customizer  |
 | ------------- |:-------------| :-----|
-| Conditional formatting based on item values and value ranges      | Supported | Supported |
+| https://www.youtube.com/watch?v=AmC9SmCBUj4ional formatting based on item values and value ranges      | Supported | Supported |
```
Comment (Lauragra) [inline docs/declarative-customization/column-formatting.md]: Did you mean to insert a link here?

Comment (VesaJuvonen) [reply docs/declarative-customization/column-formatting.md]: Definitely should not be there... even though Gordon is entertaining, but not in this context.

**Comment by VesaJuvonen** [comment]
Closing this one, since it clearly has a mistaken edit in the submission. If the question marks should be added, let's submit a new PR rather than merge this.

**Comment by ldemaris** [comment]
Yeah.  I messed that one up.  That’s what I get for doing this hastily at night.

I admittedly don’t know how to submit a PR here.

**Comment by betrue-final-final** [comment]
Lincoln,
If you need some help, let me know.

**Comment by ldemaris** [comment]
Thanks, everybody, for catching my mistakes and helping out.

The changes are simple:


  *   Add “?” to the list of operators in the “Operators” section
  *   In the “Conditional operators” section, change

The conditional operator is:
·       :

To

The conditional operator is:
·       :
Or
·       ?

**Pull request closed without merging**